content/posts/2020-09-08-analiza-wynikow/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Smutne wyniki testu C++"
+subtitle: "Dwa lata później"
```

Comment:
Czy subtitle wyświetla się też na stronie głównej, czy tylko na stronie posty? Jeśli nie wyświetla się na głównej to zróbmy tak:
```suggestion
title: "Smutne wyniki testu C++ dwa lata później"
subtitle: ""
```

Reply:
U mnie jak odpaliłem jest tylko na stronie postu.